core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -50,7 +50,7 @@ <h3>Upload Thumbnail</h3>
             <i class="material-icons oppia-vcenter">&#xE14C;</i>
           </button>
           <div>
-            <img ng-if="uploadedImage" class="oppia-thumbnail-image" ng-src="<[uploadedImage]>" height="<[aspectRatio === '4:3' ? '186px' : '180px']>" width="<[aspectRatio === '4:3' ? '248px' : '320px']>">
+            <oppia-svg-thumbnail-display ng-if="uploadedImage" [classes]="['oppia-thumbnail-image']" [img-src]="uploadedImage" [height]="aspectRatio === '4:3' ? '186px' : '180px'" [width]="aspectRatio === '4:3' ? '248px' : '320px'" [background]="bgColor"></oppia-svg-thumbnail-display>
```

Comment:
For height and width, why don't we just pass `aspectRatio` and compute height and width inside the component controller? WIll that be simpler?

Reply:
It will be. Changed!